src/components/Data/Layout.vue

```diff
@@ -71,4 +71,27 @@ export default {
   overflow: auto;
   padding: $content-gutter;
 }
+
+.Custom-resizer > .multipane-resizer {
+  margin: 0; left: 0;
+  position: relative;
+  padding: 3px;
+  border: 1px solid #ccc;
```

Comment:
Would be great to move the box-shadow from the wrapper to the resizer

```suggestion
  border: 1px solid #ccc;
  box-shadow: 0px 0px 5px 0px rgba(112, 112, 112, 1);
```